src/components/Proposal/VoteBox.tsx

```diff
@@ -37,7 +38,8 @@ export default class VoteBox extends React.Component<IProps, IState> {
   }
 
   public render() {
-    const { currentVote, proposal, daoTotalReputation, transactionState } = this.props;
+    const { currentVote, voterReputation, proposal, daoTotalReputation, transactionState } = this.props;
+    console.log('voterReputation', voterReputation)
```

Comment:
remove console

Reply:
removed + banned it on the linter (console.error are still allowed): https://github.com/daostack/alchemy/pull/39/commits/dfe12d948ae968487363cb9fdfef5c5a647eb03e